src/storage/txn/commands/flashback_to_version.rs

```diff
@@ -71,23 +71,20 @@ impl CommandExt for FlashbackToVersion {
 
 impl<S: Snapshot, L: LockManager> WriteCommand<S, L> for FlashbackToVersion {
     fn process_write(mut self, snapshot: S, context: WriteContext<'_, L>) -> Result<WriteResult> {
-        let mut reader = ReaderWithStats::new(
```

Comment:
One of the main reasons we use `SnapshotReader` here is that `ReaderWithStats::new` returns a `SnapshotReader` wrapped with `context.statistics`. If we want to use `MvccReader`, the statics info needs to be handled manually by ourselves.

Reply:
Put them both in the reader to solve this problem.
for `process_write` it will be written in drop.
for `process_read`it will be handled manually.